.github/workflows/cli-coexistence_tests.yaml

```diff
@@ -14,6 +14,32 @@ jobs:
       - name: Checkout code
         uses: actions/checkout@v1
 
+      - name: Set up Go 1.x
+          uses: actions/setup-go@v3
```

Comment:
The indentation of this section is not quite right.

Reply:
Yes, already checking it. Thanks